assets/js/vue-apps/components/grants-back-to-search.vue

```diff
@@ -0,0 +1,46 @@
+<script>
+import { storageAvailable, getWithExpiry } from '../../helpers/storage';
+import IconArrowLeft from './icon-arrow-left.vue';
+const canStore = storageAvailable('localStorage');
+
+export default {
+    components: { IconArrowLeft },
+    props: ['label', 'storageKey', 'prefix'],
+    data() {
+        return {
+            returnLink: undefined
+        };
+    },
+    computed: {
+        id() {
```

Comment:
Don't worry about this for now, but I reckon we should push this unique id logic into the icon component (happy to pick that up)